q2_assembly/quast/quast.py

```diff
@@ -382,4 +412,34 @@ def evaluate_contigs(
         # 3. read it as a pandas dataframe then we create the QUASTResults
         tabular_results = ctx.make_artifact("QUASTResults", report_df)
 
-    return tabular_results, visualization
+        # 4. create the Genomes
+        if not references:
+            try:
+                genomes = ctx.make_artifact("GenomeData[DNASequence]", genomes_dir)
+            except ValidationError as e:  # corrupt files
+                if "Missing one or more" in str(e):
+                    warn(
+                        "WARNING: QUAST did not download any genomes. The returned "
+                        "GenomeData[DNASequence] artifact is empty. Please check "
+                        "the network connection or provide the reference genomes."
+                    )
+                else:
+                    warn(
+                        "WARNING: There is a problem with the downloaded genome files. "
+                        "The returned GenomeData[DNASequence] artifact will be empty. "
+                        "You can either try again or provide the reference genomes."
+                    )
+                    # we also need to remove whatever is in the directory that
+                    # will not comply with the GenomeSequencesDirectoryFormat
+                    for f in os.listdir(genomes_dir.path):
+                        fp = os.path.join(genomes_dir.path, f)
+                        if os.path.isdir(fp):
+                            shutil.rmtree(fp)
+                        else:
+                            os.remove(fp)
```

Comment:
Honestly, instead of doing all this, we can also just re-create the genomes_dir (the old one will be cleaned up when the action exits):
```
genomes_dir = GenomeSequencesDirectoryFormat()
```